frontend/appflowy_flutter/lib/user/application/reminder/reminder_bloc.dart

```diff
@@ -512,19 +496,11 @@ class ReminderBloc extends Bloc<ReminderEvent, ReminderState> {
   ) async {
     final List<ReminderPB> availableReminders = [];
     final reminderIds = reminders.map((e) => e.id).toSet();
-    final removedIds = <String>{};
```

Comment:
If we delete the `removedIds`, how do we perform the unreferenced reminder now?

Reply:
putting removeIds here is not a correct way, however, I haven't thought of the most appropriate way to handle it

we'll ignore the unreferenced reminder now